src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/CalculateSinglePayment.java

```diff
@@ -39,7 +39,7 @@
    */
   @RequestMapping("/getSinglePaymentURL")
   public SinglePaymentResponse singlepaymenturl(
-		  @RequestParam(value="transType", required = true) String transType,
+		/*  @RequestParam(value="transType", required = true) String transType,
 		  @RequestParam(value="invoiceNumber", required = true) String invoiceNumber,
 		  @RequestParam(value="approvedPage", required = true) String approvedPage,
 		  @RequestParam(value="declinedPage", required = true) String declinedPage,
```

Comment:
why are those lines commented?

Reply:
removed the comment. Those @RequestParams are required